moe.karaokes.mugen.app.yml

```diff
@@ -0,0 +1,93 @@
+app-id: moe.karaokes.mugen.app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+command: km-wrapper
+separate-locales: false
+finish-args:
+  - '--filesystem=host'
```

Comment:
that's a very broad file system permission.

Reply:
We debated with that internally. The app stores its data by default in `home` because it needs to make some files available for streamers (text files containing info they can display on their stream's layout via OBS for example). Since it can download and manipulate big files, we also have an option to let users store their video folder on something other than `home`, hence the `host` permission.

If there's a better way to do this permission-wise (allow home as well as any removeable media like an external HDD) I'd like to know to modify the yaml accordingly.

Reply:
We've found a way to limit the filesystem permissions to `~KaraokeMugen:create`, we need to release a new version though to fix that. I'll resolve this conversation once this is done.

Reply:
Thanks to @Dragicafit this should be better now. See [this commit](https://github.com/flathub/flathub/pull/3670/commits/288e9ffe66482eaafc415f09c592f3d1c43f657f)